lib/commons/text/get-line-count.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Return the number of line wraps in a DOM element
+ * 
+ * IMPORTANT: This function assumes single-column flowing text.
+ * Floating images, flex-box, and positioned content can throw off
+ * the count. 
+ * @param {Element} domNode 
+ * @param {number} margin (default: 2)
+ * @returns {number}
+ */
+export default function getLineCount(domNode, margin = 2) {
```

Comment:
This function could be simplified to something like `isMultilineText` and then return if `range.getClientRects() > 1`. Calculating the exact number of lines is not necessary and could be error prone. All we really need to know is if there is more than 1 line.